### DISCUSSION TITLE
yarn create next-app with --example flag creates an empty project

### DISCUSSION THREAD

**Discussion by doougui** [vercel/next.js#38795] [Help]
Hello. I'm trying to create a new next app based on a defined boilerplate that I have, but for some reason it is creating an empty repository with nothing but the `.git` folder.

Basically, when I run 

`yarn create next-app --example "https://github.com/doougui/next-boillerplate" douglasgoulart.com` (The [template repository](https://github.com/doougui/next-boillerplate) is public).

I get this:

![image](https://user-images.githubusercontent.com/44846329/179771850-eb599d12-36d5-41a9-a93f-bd0e249ddddb.png)

But when I `cd douglasgoulart.com`, I get nothing:

![image](https://user-images.githubusercontent.com/44846329/179771948-b235a575-0718-4eba-8b7c-8d8bbe8ccec4.png)

I thought it might be something related to the Next.js version, but I ran 'yarn upgrade' and installed the latest version of `next` and it's still not working. 

Any ideas?

EDIT: Btw, it was working before. I have already used this same command with the same template for [another project](https://github.com/Academunity/web) not so long ago (1 month ago or so).

**Comment by valcosmos** [comment]
Hello, I tried, but it does work.

cmd: `yarn create next-app --example "https://github.com/doougui/next-boilerplate" douglasgoulart.com`

![image](https://user-images.githubusercontent.com/53572196/179882509-6b8bd019-8ea2-45d7-9061-3c4e80ea2dbd.png)

**Reply by doougui** [reply]
Hmmmm that's really weird. Two more friends of mine also tried it and it didn't work :(

**Reply by doougui** [reply]
Oh yeah, it's something related to our environment. I tried it on my MacOS work device and it worked normally. However, in my friend's MacOS work device it didn't. Now I need to find out why...

**Reply by valcosmos** [reply]
Hello,  actually, I also failed the first time I tried, but every time after that it was successful, could it be a network issue?
